### PR TITLE
IX Bid Adapter: prefer pageUrl to refererUrl

### PR DESCRIPTION
**Pull request by lksharma**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
Prioritizes pageUrl set via setConfig() over default document referrer

### CODE REVIEW

File: modules/ixBidAdapter.js [hunk 1]
```diff
@@ -591,7 +591,7 @@ function buildRequest(validBidRequests, bidderRequest, impressions, version) {
     }
 
     if (bidderRequest.refererInfo) {
-      r.site.page = bidderRequest.refererInfo.referer;
+      r.site.page = config.getConfig('pageUrl') || bidderRequest.refererInfo.referer;
```
Comment (patmmccann) [inline modules/ixBidAdapter.js:594]: do you want to sanity check values?

**Review by patmmccann — CHANGES_REQUESTED** [review]
Should you sanity check values? Why is pageUrl conditional on this other object?

File: modules/ixBidAdapter.js (hunk 1 above)
Comment (lksharma) [reply modules/ixBidAdapter.js]: Good point. I've made changes to set `site.page` only if pageUrl is valid.

**Comment by lksharma** [comment]
@patmmccann,  I've made changes to set `site.page` only if pageUrl is valid. Let me know if there is anything else that may need updating.

**Comment by lksharma** [comment]
@patmmccann, is there anything else pending for the review?

**Comment by patmmccann** [comment]
@lksharma sorry for the delay here? It seems like these changes in #8059 include this?

**Comment by patmmccann** [comment]
This is ready to merge if you want

**Comment by lksharma** [comment]
Hey @patmmccann, 

I'll be closing this MR. I've includes these changes in another open MR here: https://github.com/prebid/Prebid.js/pull/8059

**Pull request closed without merging**